table.go

```diff
@@ -653,6 +662,7 @@ func (t *Table[T]) DefaultMouseDown(where Point, button, clickCount int, mod Mod
 	if t.Window().InDrag() {
 		return false
 	}
+	t.wasDragged = false
 	t.RequestFocus()
 	t.interactionRow = -1
 	t.interactionColumn = -1
```

Comment:
... and then do a `t.lastSel = zeroUUID` here... and please move the `t.wasDragged = false` to after the `t.RequestFocus()` line.